### PR TITLE
Removes inline css from all pages

### LINKED ISSUE #990: SASS optimizations
- [ ] Remove redundancy by using SASS mixins
- [ ] Adding repeated values like padding, margin, colors etc to config.scss
- [x] Removing inline CSS from handlebars and adding it to SASS.

### PR DESCRIPTION
**Pull request by geekyd**
solves [c] #990 

Changes: Removes inline css from all the pages and adds to application.scss.

Demo:
Server: https://open-event-web-appp.herokuapp.com
gh-pages: https://geekyd.github.io/eventApplication/

@mariobehling @aayusharora @Princu7 Please review. Thanks

### CODE REVIEW

File: src/backend/_scss/application.scss [hunk 1]
```diff
@@ -347,6 +347,15 @@ a {
   max-height: 120px;
 }
 
+.sponsor-container {
+  background: $white;
+}
+
+.sponsor-row {
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: Can you please right margin  in single line.

Comment (geekyd) [inline src/backend/_scss/application.scss]: Sure, any other changes you want me to do?

**Comment by geekyd** [comment]
@aayusharora @Princu7 I have made the requested changes, please review. Thanks

File: src/backend/_scss/application.scss (hunk 1 above)
Comment (aayusharora) [inline src/backend/_scss/application.scss]: No, it looks good apart from that.

**Comment by aayusharora** [comment]
@geekyd it does not completely fix the issue, it only solves the last point of the issue.
Please use "Fix  #issue no"  carefully, as it closes the issue

**Comment by geekyd** [comment]
@aayusharora I have changed the description, the issue won't close on merge. Please merge this. Thanks

**Comment by geekyd** [comment]
@aayusharora @Princu7 Please review this for merge. Thanks

**Pull request merged**